deploy/templates/envoy-template.yml

```diff
@@ -0,0 +1,252 @@
+# This template creates the configmap that contains the configuration of the
```

Comment:
IN app-sre we are deploying backend just fomr `service-template.yml`, I think we have to move this content to that file as well.

Reply:
yes. This configmap needs to be there also